src/libraries/System.Text.Encoding/tests/Ascii/EqualsTests.cs

```diff
@@ -55,15 +75,18 @@ public static IEnumerable<object[]> DifferentInputs
             {
                 yield return new object[] { "tak", "nie" };
 
-                for (char i = (char)1; i <= 127; i++)
+                foreach (int textLength in BufferLengths)
                 {
-                    if (i != '?') // ASCIIEncoding maps invalid ASCII to ?
+                    for (char chr = (char)0; chr <= 127; chr++)
                     {
-                        yield return new object[] { new string(i, i), string.Create(i, i, (destination, iteration) =>
+                        if (chr != '?')
```

Comment:
this comment was valuable, please restore it

```suggestion
                        if (chr != '?') // ASCIIEncoding maps invalid ASCII to ?
```